packer-build/ubuntu1804.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Does packer support YAML, or better yet, HCL? It might make for a more readable file, esp because you can add comments.

Reply:
None.